src/components/UserList/Row.js

```diff
@@ -147,10 +149,28 @@ class Row extends Component {
       </View>
     );
   }
+  
+  onRequestCallback(result, error) {
+    if (error) {
+      Alert.alert('Error on Bookmark', error);
+    } else if (result) {
+      this.forceUpdate();
+    }
+  }
 
   sendRequest() {
     Actions.requestPage({ id: this.props.dataSource._id, me: this.props.dataSource.me });
   }
+  
+  setBookmark() {
+    if (this.state.bookmarked) {
+      UserUtil.bookmarkOff(this.onRequestCallback.bind(this), this.props.dataSource._id);
+      this.state.bookmarked = false;
```

Comment:
Yes, just for image temporarily but after refresh the state would be same as before.